include/tvm/ir/type.h

```diff
@@ -312,5 +312,26 @@ class FuncType : public Type {
   TVM_DEFINE_OBJECT_REF_METHODS(FuncType, Type, FuncTypeNode);
 };
 
+class TensorMapTypeNode : public TypeNode {
+ public:
+  void VisitAttrs(AttrVisitor* v) { v->Visit("span", &span); }
```

Comment:
does not have to be now, see https://github.com/apache/tvm/pull/18098 to transition to new reflection, i can do that after erge as well